expansion/src/main/java/no/unit/nva/expansion/utils/ExpandedTicketStatusQueries.java

```diff
@@ -0,0 +1,40 @@
+package no.unit.nva.expansion.utils;
+
+import no.unit.nva.expansion.model.ExpandedTicketStatus;
+import no.unit.nva.publication.model.business.TicketEntry;
+import no.unit.nva.publication.model.business.TicketStatus;
+
+import java.util.Objects;
+
+public class ExpandedTicketStatusQueries {
```

Comment:
I think I would rename this class to ExpandedTicketStatusMapper. "Queries" is not describing here.